src/components/modalBreadcrumb/modalBreadcrumb.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <nav class="apos-breadcrumb" aria-label="breadcrumb">
+    <ol class="apos-breadcrumb__items">
+      <li v-for="(item, index) in items" :class="`apos-breadcrumb__item ${modifier}`">
+        <component :is="item.href ? 'a' : 'span'" :href="item.href">{{ item.label }}</component>
+        <ChevronRightIcon class="apos-breadcrumb__chevron" :size="13" v-if="index !== last" />
+      </li>
+    </ol>
+  </nav>
+</template>
+
+<script>
+import ChevronRightIcon from 'vue-material-design-icons/ChevronRight.vue';
+export default {
+  props: {
+    label: {
+      default: 'Set a label',
+      type: String
+    },
+    modifier: {
+      default: '',
+      type: String
+    },
+    items: {
+      default: [],
+      type: Array
+    },
+  },
+  computed: {
+    last() {
+       return Object.keys(this.items).length - 1;
+    }
+  },
+  components: {
+    ChevronRightIcon
+  }
+}
+</script>
+
+<style lang="scss">
+  .apos-breadcrumb__items {
+    margin-block-start: 0;
```

Comment:
TIL.

This and `padding-inline-start` seem too unsupported. Are we going to need this vertical as well? If so, maybe it should get a prop passed in for direction instead, which could manage padding.